### PR TITLE
Add support for DS Energy resources

### PR DESCRIPTION
**Pull request by Raphexion**
Add initial support for DS Energy.

### CODE REVIEW

File: lib/wise_homex/models/ds_energy_location.ex [hunk 1]
```diff
@@ -0,0 +1,22 @@
+defmodule WiseHomex.DSEnergyLocation do
+  @moduledoc false
+
+  use WiseHomex.BaseModel
+
+  embedded_schema do
+    belongs_to :ds_energy_group, WiseHomex.DSEnergyGroup
+    belongs_to :household, WiseHomex.Household
+    belongs_to :room, WiseHomex.Room
```
Comment (lasseebert) [inline lib/wise_homex/models/ds_energy_location.ex:9]: Household and room are not exposed in the API

File: lib/wise_homex/models/ds_energy_location.ex [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    field :customer_name, :string
+    field :location_name, :string
+    field :full_location, :string
+    field :full_location_name, :string
+
+    field :parent_full_location, :string
+
+    field :depth, :integer
```
Comment (lasseebert) [inline lib/wise_homex/models/ds_energy_location.ex:18]: Depth is not exposed in the API

File: lib/wise_homex/models/ds_energy_meter.ex [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    belongs_to :ds_energy_location, WiseHomex.DSEnergyLocation
+
+    field :meter_id, :string
+    field :friendly_name, :string
+    field :meter_type, :string
+    field :comment, :string
+    field :alarm_email, :string
+    field :location_name, :string
+    field :full_location_name, :string
+    field :full_location_id, :string
```
Comment (lasseebert) [inline lib/wise_homex/models/ds_energy_meter.ex:16]: This is called `full_location` in the API

File: lib/wise_homex/models/household.ex [hunk 4]
```diff
@@ -13,6 +13,7 @@ defmodule WiseHomex.Household do
     has_many :settlement_values, WiseHomex.SettlementValue
     has_many :unsuccessful_visits, WiseHomex.UnsuccessfulVisit
     has_one :ready_installation, WiseHomex.ReadyInstallation
+    has_one :ds_energy_location, WiseHomex.DSEnergyLocation
```
Comment (lasseebert) [inline lib/wise_homex/models/household.ex:16]: This relation is not exposed in the API

**Review by lasseebert — CHANGES_REQUESTED** [review]
It seems that dome things are not consistent with the API (see my comments).
Is there something in the API that is not yet committed?

File: lib/wise_homex/models/ds_energy_location.ex (hunk 1 above)
Comment (Raphexion) [reply lib/wise_homex/models/ds_energy_location.ex:9]: Good catch. Sorry about that. Now exposed in the API :sunny:

File: lib/wise_homex/models/ds_energy_location.ex (hunk 2 above)
Comment (Raphexion) [reply lib/wise_homex/models/ds_energy_location.ex:18]: Good catch. Sorry about that. Now exposed in the API :sunny:

File: lib/wise_homex/models/household.ex (hunk 4 above)
Comment (Raphexion) [reply lib/wise_homex/models/household.ex:16]: Good catch. Sorry about that. Now exposed in the API :sunny:

File: lib/wise_homex/models/ds_energy_meter.ex (hunk 3 above)
Comment (Raphexion) [reply lib/wise_homex/models/ds_energy_meter.ex]: Good catch :muscle: Fixed and squshed :sunny:

**Review by lasseebert — APPROVED** [review]
Super :tada:

**Pull request merged**